### ISSUE TITLE
This project needs new maintainers 🚨 Apply here!

### ISSUE THREAD

**Issue by tomav** [tomav/docker-mailserver#1186]
Dear all,

When I published this project, I did not expect that so many people were interested in it.
I'm happy to see that this project reached more than 4000+ Github stars and 700+ forks.
This was possible thank to this community, a lot was brought by you guys, and I'd like to thank all of you for this. 👍
Special thanks to @johansmitsnl who did a great job as a maintainer on this project these last 2 years but recently decided to move to something else.

Being myself currently focused on other priorities, **this project is now looking for 3 maintainers**.

If you have an experience managing this kind of project on Github, you can apply here.
The goal will be to:
- review and merge Pull Requests
- fix the CI, integrations tests to bring back the green status (some optimization can be done here)
- upgrade the underlying stack
- define and share a strategy to this repo

If you have question, we can discuss here.

Thomas

**Comment by erik-wramner** [comment]
I've already fixed the tests, see pull requests.

I haven't managed any projects on Github, but I do have 25+ years as a professional developer and I'm somewhat active in open source, so I think I can handle that. Unfortunately I don't have much free time, so if there are others who are willing to dedicate more time I may not be the best choice.

As for strategy I think it is important that the maintainers have a shared view. Personally I like the "fullstack but simple mailserver" with "only configuration files, no SQL" mission statement. I would prefer to have a stack that is secure, very easy to configure and maintain and with as few moving parts (that can and will break!) as possible. Stability, security and ease of maintenance rather than features in other words.

That strategy is important for pull requests; I would be reluctant to add features unless they are really needed for delivering mail, but would be eager to add bug fixes and security improvements.

Technically I'm more of a developer than a Linux or Postfix specialist. I have worked with Linux for a long time and I do have a few mail servers that I maintain, but it has never been my full-time job.

In short I can step up as a maintainer if the community thinks I'm right for the job given the disclaimer about limited time and my vision for where this should be going (and not!). Otherwise I will be happy to help by submitting pull requests instead.

**Comment by martin-schulze-vireso** [comment]
I'm afraid this will sound familiar but here I go:

I only have experience in smaller projects but +10 years of professional software development. I have made small contributions to many different projects but my free time is very limited.

I have not done much with the code of this project so far, so I would need to get my feet wet first. I would be willing to do some of the ground work and help in discussions before laying hand onto larger changes.

Like @erik-wramner I manage some mail servers and use Linux but postfix is not part of my full-time job as software engineer.

I'd have to dive deeper to form a strategy but I can second that I favor stability over new features. I usually have a good eye for what needs to be done and have a "if I don't do it, nobody does it" mentality. In the short term I would want to get the tests green again and weed out the current PRs and issues.

So in summary: I would be willing to help as a co-maintainer but I don't think I am up to the task of being the lead (for lack of time and experience). I'd stand in line behind @erik-wramner as I believe he is more experienced.

**Comment by fbartels** [comment]
Hi,

I am a "downstream" user of this project and am integrating it in https://github.com/zokradonh/kopano-docker. Here I have already experimented quite a bit with Travis CI and integration into the Docker hub (at work we are using private instances of Jenkins so that is not directly applicable here).

I would like to also work a bit on updating and minimising the current stack, as the system how it is at the moment is quite heavy.

**Comment by martin-schulze-vireso** [comment]
Anyone of the recent/prolific contributors? What about you @danielpanteleit @m-schmoock @501st-alpha1 @phish108 @17Halbe @jurekbarth @TechnicLab @0xflotus @paralax

Sorry for the noise but with this large a project there have to be more than three people willing to step up. Let this be a real choice.

**Comment by m-schmoock** [comment]
@martin-schulze-vireso  Sorry, I don't have enough spare time to maintain this.

**Comment by phish108** [comment]
I really enjoy this project. Thank you @tomav! I can understand the problem of keeping a project alive if other tasks call. I am very much interested in backing any project maintainer who keeps the project running. But like @tomav I have no capacity doing it myself. 

@martin-schulze-vireso I can help with documenting and do a bit of script hacking.

**Comment by 501st-alpha1** [comment]
I'd be happy to help maintain this project.  I do have some experience maintaining other (smaller) open source projects.

I have a little over three years of professional software development experience, though it's not directly related to this, since it's mostly full stack web development.  I do run and maintain docker-mailserver instances, as well other GNU/Linux servers.

I agree with @erik-wramner's strategy view; security & stability > new features, especially for something like a mail server.

I am also lacking in free time at present, so due to that and my comparative experience, I'd be happy to be a co-maintainer, but probably shouldn't be the primary.

I would be particularly focused on triaging issues, so in order to get a feel for that I am now watching the repo, and will see how that goes.  I'm also happy to review PRs and maintain the documentation where I can.

TL;DR: I'd be happy to be a co-maintainer, with a focus on issue triage.

**Comment by jurekbarth** [comment]
@martin-schulze-vireso Sorry, i won't have enough time to manage this project on a regular basis right now.

**Comment by tomav** [comment]
@erik-wramner & @fbartels, where are you located? (country, timezone).
It might be interesting to chat about that.

**Comment by tomav** [comment]
(On a side note, thanks to @erik-wramner, tests are now green) 👍

**Comment by fbartels** [comment]
Seems we are all in europe. Biggest city next to me would be Rotterdam, NL.

**Comment by erik-wramner** [comment]
I'm in Gothenburg (CET). At present it is vacation time here, so I will be offline every now and then.

**Comment by tomav** [comment]
Thanks guys. We'll have to catchup quickly early September to talk about basic guidelines and future of DMS.
In the meantime, I'll add you as maintainer.
